src/mudlet-lua/lua/geyser/GeyserContainer.lua

```diff
@@ -190,6 +190,41 @@ function Geyser.Container:lower ()
 	lowerWindow(self.name)
 end
 
+function Geyser.Container:raiseAll(container, me)
+  me = me or true
+  container = container or self
+  -- raise myself
+  if me then 
+    container:raise() 
+  end
+  
+  local v
+  for i=1,#container.windows do
+    v = container.windows[i]
+    container.windowList[v]:raise()
+    container.windowList[v]:raiseAll(container.windowList[v], false)
+  end
+end
+
+local function createReverseTable(container)
+ local v
+  Geyser.Container.reverseTable = Geyser.Container.reverseTable or {}
+    for i=1,#container.windows do
+    v = container.windows[i]
+    table.insert(Geyser.Container.reverseTable, container.windowList[v])
+    createReverseTable(container.windowList[v])
+  end
+end
+
+function Geyser.Container:lowerAll()
+  createReverseTable(self)
+  for i=#Geyser.Container.reverseTable,1,-1 do
```

Comment:
We create a reverse table that we then iterate backwards? That's strange - why not create a normal table that we iterate as usual then?

Looking at the code the table seems to be normal one that we iterate in reverse, right?

(let me know if this explanation is confusing)

Reply:
yeah, that is exactly what's happening. Probably the name reverseTable is a bit misleading :wink: